### DISCUSSION TITLE
Quick little GUI to edit the config file

### DISCUSSION THREAD

**Discussion by ofernander** [Louisvdw/dbus-serialbattery#817] [General]
https://github.com/0x0f906/dbus-serialbattery-gui

Quick and dirty, welcome anyone who wants to expand upon it.

**Comment by mr-manuel** [comment]
You could add some screenshots :-)

**Comment by ofernander** [comment]
Haha sorry! For anyone interested, working on the ability to display and also tail log files right now. 

![gui](https://github.com/Louisvdw/dbus-serialbattery/assets/35053416/aaf67faa-f841-4583-9412-82d4565ff0e4)

**Reply by mr-manuel** [reply]
I see nowhere that you call the `reinstall-local.sh`. If you only push the `config.ini` to remote and restart the driver, the changes will not be applied. After changing the config you have to execute `reinstall-local.sh`. You could add a button `Apply Remote Config` or add this action when pressing `Push Config to Remote`.

**Reply by ofernander** [reply]
I've been changing the config.ini file and calling the restart script and it's been applying the changes. I have not been utilizing the reinstall function. 

It's a good idea to change "Restart dbus-serialbattery" to "Apply Config" and give the user a warning that it will restart the driver. I will make those changes.

**Reply by mr-manuel** [reply]
You are right, I forgot about the restart script. It also copies the new config file.
